src/codegen/doc/mod.rs

```diff
@@ -766,11 +765,11 @@ fn get_type_trait_for_implements(env: &Env, tid: TypeId) -> String {
         format!("{}Ext", env.library.type_(tid).get_name())
     };
     if tid.ns_id == MAIN_NAMESPACE {
-        format!("[`trait@crate::prelude::{}`]", &trait_name)
+        format!("[{name}](crate::prelude::{name})", name = &trait_name)
```

Comment:
That's true. The change can be reverted although, a huge reason why [][] wasn't a great choice is because of the broken links which we mostly don't have with this approach.

Reply:
`[][]` is still probably the better choice, so that a broken link is rendered as the plaintext markdown instead of being clickable and linking to garbage. Let's discuss this elsewhere though :)

Reply:
Just noticed, `trait@` got lost here too.